### PR TITLE
Login

### PR DESCRIPTION
**Pull request by cer-nagas**
Add a LOGIN tab on Navigation Bar, which opens a simple Login page on a new tab.

### CODE REVIEW

File: frontsite/views.py [hunk 1]
```diff
@@ -19,3 +23,35 @@ def index(request):
             'form': form,
             'job': job
         })
+
+def user_login(request):
+    template = 'login.html'
+
+    if request.method == 'POST':
+        # Gather the username and password provided by the user.
```
Comment (ngdangdat) [inline frontsite/views.py]: Please remove redundant comments. Re-read your written code, comment lines that you find out it's difficult to understand as a code reader. For example, `username = request.POST.get('username')`, this line has a clear meaning that you want to get username key from POST dict => comment is redundant here.

Comment (cer-nagas) [reply frontsite/views.py]: The comment is not about what that line of code does. It's a note about the reason we're choosing request.POST.get() over request.POST. Thus I found it necessary.

File: frontsite/views.py [hunk 2]
```diff
@@ -19,3 +23,36 @@ def index(request):
...
+        user = authenticate(username=username, password=password)
+        # If we have a User object, the details are correct.
+        if user:
+            if user.is_active:
+                login(request, user)
+                return render(request, template, {'message': "You are logged in successfully!"})
+            else:
+                return render(request, template, {'message': "Your account is disabled."})
+        else:
+            print("Invalid login details: {0}, {1}".format(username, password))
```
Comment (ngdangdat) [inline frontsite/views.py]: Please remove `print` here. It's for debugging in development, not production environment.

File: frontsite/views.py [hunk 3]
```diff
@@ -19,3 +23,36 @@ def index(request):
...
+            # # Uncomment to create a dummy user and print all users on webpage
```
Comment (ngdangdat) [inline frontsite/views.py]: What is the purpose of these commented lines?

File: frontsite/views.py (hunk 1 above)
Comment (ngdangdat) [reply frontsite/views.py]: `request.POST` is a dict in Python. The purpose of this class's `get` method is helping developer to control returned value better. The note is already there when you type `.get(<key>)` after `request.POST`, which makes I find these comments redundant.

**Pull request merged**